Libplanet.Tests/Action/AccountStateDeltaTest.cs

```diff
@@ -354,9 +354,23 @@ public virtual void BurnAsset()
                 0
             );
 
-        protected FungibleAssetValue? GetTotalSupply(Currency currency) =>
-            _totalSupplies.TryGetValue(currency, out var totalSupply)
-                ? new FungibleAssetValue(currency, totalSupply.Item1, totalSupply.Item2)
-                : (FungibleAssetValue?)null;
+        protected FungibleAssetValue GetTotalSupply(Currency currency)
+        {
+            if (!currency.TotalSupplyTrackable)
+            {
+                var msg =
+                    $"The total supply value of the currency {currency} is not trackable because it"
+                    + " is a legacy untracked currency which might have been established before"
+                    + " the introduction of total supply tracking support.";
+                throw new TotalSupplyNotTrackableException(currency, msg);
```

Comment:
The same exception message seems repeated.  How about making an internal static method like `TotalSupplyNotTrackableException.WithDefaultMessage()`?